src/ClaimService.Business/Features/Categories/Commands/Create/CreateCategoryValidator.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.ClaimService.Business.Features.Claims.Commands.Create;
+
+namespace LT.DigitalOffice.ClaimService.Validation.Claim;
+
+public class CreateCategoryValidator : AbstractValidator<CreateCategoryCommand>
```

Comment:
where are you calling the validator?

Reply:
good notice also forgot to add validation pipeline